webapp/src/Controller/Team/MiscController.php

```diff
@@ -103,8 +103,7 @@ public function homeAction(Request $request): Response
                 ->leftJoin('c.sender', 's')
                 ->leftJoin('c.recipient', 'r')
                 ->select('c', 'cp', 'p')
-                ->andWhere('c.contest = :contest')
-                ->andWhere('cp.contest = :contest')
+                ->andWhere('c.contest = :contest OR cp.contest = :contest')
```

Comment:
General question about the existing code:
When would ever `cp.contest` match `$contest`, but `c.contest` not? That is, why can't we just have
```
->andWhere('c.contest = :contest')
```

Reply:
When you unlink a problem from a contest is the only way I can think of.

Reply:
this was implemented in 0219d8e173540e26b5867d219753dcb23f08fcc5